src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --navigation-link-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+  font-size: 12px;
+  min-width: 1024px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  justify-content: flex-end;
+  width: 100%;
```

Comment:
it would be better to use space-between for justify-content instead of margin-right: auto for logotype 
also you don't need width: 100%

Reply:
Adjusted